janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphOperationCountingTest.java

```diff
@@ -469,13 +471,17 @@ public void testCacheConcurrency() throws InterruptedException {
                 option(GraphDatabaseConfiguration.METRICS_PREFIX),metricsPrefix};
         clopen(newConfig);
         final String prop = "someProp";
-        makeKey(prop,Integer.class);
+        final String vertex = "someVertex";
+        VertexLabel vertexLabel = mgmt.makeVertexLabel(vertex).make();
+        PropertyKey propertyKey = makeKey(prop, Integer.class);
+        mgmt.addProperties(vertexLabel, propertyKey);
         finishSchema();
 
         final int numV = 100;
         final long[] vertexIds = new long[numV];
         for (int i=0;i<numV;i++) {
-            JanusGraphVertex v = graph.addVertex(prop,0);
```

Comment:
This got me wondering, how should we handle the case where a vertex without a label is added? Should someone be able to specify mappings for the no vertex label case? Running a quick test and a constraint violation isn't thrown for this:
```
gremlin> g.addV().property('name', 'foo')
==>v[4272]
```

Reply:
You add a vertex with the label `vertex`.